titus-common/src/main/java/com/netflix/titus/common/framework/scheduler/model/TransactionId.java

```diff
@@ -0,0 +1,70 @@
+package com.netflix.titus.common.framework.scheduler.model;
+
+import java.util.Objects;
+
+public class TransactionId {
+
+    private static final TransactionId INITIAL = new TransactionId(1, 0, 1);
+
+    private final int major;
```

Comment:
I think the current object model is confusing. Talked offline for other approaches.

Reply:
Changed